energyplus_regressions/diffs/table_diff.py

```diff
@@ -467,6 +467,8 @@ def table_diff(
         if uheading1 not in uhset_match:
             table_not_in_2 = 1
             count_of_not_in_2 += table_not_in_2
+            table_big_diff = 1
+            count_of_big_diff += table_big_diff
```

Comment:
A table that is missing in the mod branch is now going to also report a table big diff.